test/integration/helpers/mswjs/browserWorker.js

```diff
@@ -12,29 +12,33 @@ governing permissions and limitations under the License.
 import { setupWorker } from "msw/browser";
 import { networkRecorder } from "./networkRecorder.js";
 
-const worker = setupWorker();
+const createWorker = () => {
```

Comment:
This was the first time when I had 3 tests in one file. The network calls were captured just for the first test. Seems if I setup the worker at the beginning of each test, this issue is solved.